prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,58 @@
+"""
+CP1404 Practical- 09
+Ashton Jack Stewart
+Taxi simulator
+"""
+
+
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
+
+def main():
+    """Sets an initial set of values for the class and functions to use"""
+    print("Lets drive!")
+    taxis = [Taxi("Prius", 100),
+             SilverServiceTaxi("Limo", 100, 2),
+             SilverServiceTaxi("Hummer", 200, 4)]
+    menu = "q)uit, c)hoose taxi, d)rive"
+    current_taxi = "not chosen"
```

Comment:
A very peculiar variable

"""
A good option is to maintain something like a current_taxi... but what will the initial value of this variable be?
When you want a default starting value for an object, don't use a string or other unrelated type...
Instead, you use None:
"""
current_taxi = None